### PR TITLE
Rollback to Python 3.6 & Add 3.6 to gh actions

### PR DESCRIPTION
**Pull request by natct10**
For colab compatibility

### CODE REVIEW

**Review by arthurPignet — APPROVED** [review]
I think this release note will not be too long to write

File: .github/workflows/main_ci.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest # We could run on other OS but we would have to change the command to install libsndfile
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.6, 3.8]
```
Comment (RomainGoussault) [inline .github/workflows/main_ci.yml:11]: no 3.7 ?

Comment (arthurPignet) [reply .github/workflows/main_ci.yml:11]: We already test the 3.6 and the 3.8, I don't think it's mandatory.
This way the idea is to test min and max supported python's version

Comment (natct10) [reply .github/workflows/main_ci.yml:11]: Yes, that was the idea. Are you okay with that @RomainGoussault?

Comment (RomainGoussault) [reply .github/workflows/main_ci.yml:11]: Well it's free, so I don't see why we should not test 3.7 :)

Comment (arthurPignet) [reply .github/workflows/main_ci.yml:11]: It's a plus 50 percent on the total computation cost so it's not really free.

**Review by bowni — APPROVED** [review]

File: .github/workflows/main_ci.yml (hunk 1 above)
Comment (RomainGoussault) [reply .github/workflows/main_ci.yml:11]: By free I meant that all the builds are done in parallel so it does not have any impact on the build time.

Comment (natct10) [reply .github/workflows/main_ci.yml]: Ok added

**Pull request merged**